pkgs/data/themes/chili/plasma-chili.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "plasma-chili";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
+    owner = "MarianArlt";
+    repo = "kde-plasma-chili";
+    rev = "${version}";
+    sha256 = "17pkxpk4lfgm14yfwg6rw6zrkdpxilzv90s48s2hsicgl3vmyr3x";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/sddm/themes/plasma-chili
+    mv * $out/share/sddm/themes/plasma-chili/
+  '';
+
+  meta = with stdenv.lib; {
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
The source files mention that later versions are okay.